### PR TITLE
implements MVC pattern, adds topics endpoint

### PR DESCRIPTION
**Pull request by jbhall4291**
this is for task 3. GET /api/topics

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/topics", () => {
+  test("Status 200: the topics endpoint exists and responds", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
```
Comment (Beth-McCormack) [inline __tests__/app.test.js:21]: This test is redundant as the next tests will not pass unless status code is 200.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const app = express();
+const {getAllTopics} = require('./controllers/topicControllers.js')
+
+app.use(express.json());
```
Comment (Beth-McCormack) [inline app.js:5]: Please remove this until its needed

**Review by Beth-McCormack — CHANGES_REQUESTED** [review]
see comments

**Review by johnhutcheon — APPROVED** [review]
Thanks for making the changes Johnny, all good to merge!

**Pull request merged**